### ISSUE TITLE
Extract elementId when manipulating a cell

### ISSUE THREAD

**Issue by uriahf** [glin/reactable#375]
I wonder if there is an easy way to extract the elementId of the reactable being manipulated while writing a custom JS function.

It would be nice to have the elementId as part of the [cellInfo properties](https://glin.github.io/reactable/articles/custom-rendering.html?q=cellinfo#cellinfo-properties).

I guess an easy workaround would be just to insert the elementId in meta.state.

**Comment by glin** [comment]
Good idea! should be quite simple to do. It'd probably fit better on the table `state`. And yeah, I guess you could just use `meta` for now.
